### DISCUSSION TITLE
You have been featured in CHEPs latest videos

### DISCUSSION THREAD

**Discussion by Schnabulation** [Jyers/Marlin#483] [Q&A] [answered]
Just a quick heads up:
You have been featured in CHEPs latest YouTube videos: https://youtu.be/9TCqQF374GU

Prepare yourself for a lot more traction with your firmware... ;-)

**Comment by Reggi3** [comment, marked as answer]
been getting mentions on 'Creality Ender 3 users' and 'Marlin' discord servers as well :-)  I think smith3d are also adopting the Jyers fork too!

**Reply by darco415** [reply]
> been getting mentions on 'Creality Ender 3 users' and 'Marlin' discord servers as well :-) I think smith3d are also adopting the Jyers fork too!

wow, your right....interesting
